### DISCUSSION TITLE
Ability to cancel an active product/service

### DISCUSSION THREAD

**Discussion by realago** [Paymenter/Paymenter#315] [Ideas]
Right now, if you want to cancel a service, you either need to do it manually through the admin side (which normal users couldn't do), or wait until the month is over and let it expire. There should be a button beside the "Login to Product" named "Manage Service" which can allow you to either cancel the service right then, or at the end of the month :)

**Comment by finnjosten** [comment]
If I remember correctly, this has been suggested already and is in the works (sorry if I am wrong)

**Reply by CorwinDev** [reply]
You’re correct and will release soon
